docker/connector-db/README.MD

```diff
@@ -0,0 +1,13 @@
+# Docker Run
+
+Useful while running database connectors like mysql, mariadb, mssql
```

Comment:
@kirtimanmishrazipstack Readme file extention should be in small case instead of capital case

Reply:
@jaseemjaskp done

Reply:
@jaseemjaskp done